README.md

```diff
@@ -95,6 +96,8 @@ optional arguments:
 
 Finds and deduplicate files.
 
+**Important Warning:** This tool will only compare the property specified in the deduping method. Updating a duplicate file in a mergerfs pool will change the content of a single file but the timestamp of all copies. Running deduping with newest method under those conditions may cause data loss has the file removed will be "random".
+
```

Comment:
Why is this a warning? Why would you expect it to compare values you didn't tell it to? It is simply incorrect that updating a file in mergerfs will cause the timestamp across all to update. It depends on the settings and application's behavior.

The only thing that is perhaps unclear is that if you say "keep the newest file" and they are all the same value that it will effectively keep a random file.